README.md

```diff
@@ -30,10 +32,20 @@ config :sentry,
 ```
 
 ### Capture Exceptions
+
+Sometimes you want to capture specific exceptions, to do so use the `Sentry.capture_exception/3`.
+
 ```elixir
-Sentry.capture_exception(my_exception)
+try do
+  ThisWillError.reall()
+rescue
+  my_exception ->
+    Sentry.capture_exception(my_exception)
```

Comment:
@mitchellhenke maybe in capture_exception we should grab System.stacktrace() if its null?
https://github.com/getsentry/sentry-elixir/blob/master/lib/sentry/event.ex#L45